client/app/scripts/superdesk-content-filters/content-filters.js

```diff
@@ -425,7 +436,10 @@
 
         var fetchContentFilters = function() {
             contentFilters.getAllContentFilters().then(function(_filters) {
-                $scope.contentFilters = _filters;
+                $scope.contentFilters = _.sortBy(_filters, function(filter) {
+                    return filter.name.toLowerCase();
+                });
```

Comment:
This comparison function with identical logic is used at multiple places, IMO it should be extracted.
```js
function lowercaseName (item) {
    return item.name.toLowerCase();
}

$scope.something = _.sortBy(collection, lowercaseName);
...
$scope.somethingElse = _.sortBy(anotherCollection, lowercaseName);
...
```